Arrowgene.Ddon.GameServer/Handler/InstanceEnemyKillHandler.cs

```diff
@@ -124,6 +124,9 @@ public override void Handle(GameClient client, StructurePacket<C2SInstanceEnemyK
                 uint gainedExp = enemyKilled.GetDroppedExperience();
                 uint extraBonusExp = 0; // TODO: Figure out what this is for (gp bonus?)
 
+                uint gainedPP = enemyKilled.GetDroppedPlayPoints();
```

Comment:
Do we need to add play points to quest monsters? Or are they only for random monsters in the field?